### PR TITLE
docs: improve Edge connectivity docs

### LINKED ISSUE #960: document vertex_disjoint_paths() better
It's documented together with vertex_connectivity (I'm fixing the use of alias for that) but without many details.

### PR DESCRIPTION
**Pull request by maelle**
Fix #960

### CODE REVIEW

File: R/flow.R [hunk 1]
```diff
@@ -357,12 +357,16 @@ min_cut <- function(graph, source = NULL, target = NULL, capacity = NULL, value.
 #' called group cohesion.
 #'
 #' The vertex connectivity of two vertices (`source` and `target`) in
-#' a directed graph is the minimum number of vertices needed to remove from the
-#' graph to eliminate all (directed) paths from `source` to `target`.
+#' a directed graph is the minimum number of vertices that must be deleted to
+#'  eliminate all paths paths from `source` to `target`.
```
Comment (szhorvat) [inline R/flow.R:361]:
```suggestion
#' a graph is the minimum number of vertices that must be deleted to
#'  eliminate all (directed) paths from `source` to `target`.
```

File: R/flow.R [hunk 2]
```diff
@@ -465,18 +470,34 @@ vertex_connectivity <- function(graph, source = NULL, target = NULL, checks = TR
...
+#'
+#'
 #' A set of edge disjoint paths between two vertices is a set of paths between
 #' them containing no common edges. The maximum number of edge disjoint paths
 #' between two vertices is the same as their edge connectivity.
 #'
+#' When there are no edges from the source to the target, the number of
+#' vertex-disjoint paths is the same as the vertex connectivity of
+#' the two vertices. When some edges are present, each one of them
+#' contributes one extra path.
```
Comment (szhorvat) [inline R/flow.R:487]:
```suggestion
#' When there are no direct edges between the source and the target, the number
#' of vertex-disjoint paths is the same as the vertex connectivity of
#' the two vertices. When some edges are present, each one of them
#' contributes one extra path.
```

**Review by szhorvat — CHANGES_REQUESTED** [review]
I'd suggest the corrections I indicated. Other than that it looks good!

**Comment by maelle** [comment]
@szhorvat I'm sorry but I can't merge with Aviator if you don't review this again. :pray:

**Comment by krlmlr** [comment]
Let's open a new PR if this needs more changes.

**Comment by szhorvat** [comment]
I'm sorry, I didn't notice this! Good that it's now merged.

**Pull request merged**